ui_framework/components/info_button/info_button.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render } from 'enzyme';
+
+import {
+  KuiInfoButton,
+} from './info_button';
+
+describe('KuiInfoButton', () => {
+  describe('Baseline', () => {
+    test('is rendered', () => {
+      const $button = render(
+        <KuiInfoButton />
+      );
+
+      expect($button)
+        .toMatchSnapshot();
+    });
+
+    test('HTML attributes are rendered', () => {
+      const $button = render(
+        <KuiInfoButton
+          aria-label="aria label"
+          className="testClass1 testClass2"
+          data-test-subj="test subject string"
+          disabled
```

Comment:
This might need some more work, because `aria-label`, `data-test-subj` and `disabled` props are actually not in the "snap" thingy.